Formula/t/thors-serializer.rb

```diff
@@ -29,7 +30,9 @@ def install
 
     system "./configure", "--disable-vera",
                           "--prefix=#{prefix}",
-                          "--disable-test-with-integration"
+                          "--disable-test-with-integration",
+                          "--disable-test-with-mongo-query",
+                          "--disable-Mongo-Service"
```

Comment:
```suggestion
                          "--disable-Mongo-Service",
                          *std_configure_args.reject { |s| s["--disable-debug"] }
```

Reply:
This would not work with my config file.